setup.cfg

```diff
@@ -1,10 +1,146 @@
-[bdist_wheel]
-universal=1
+# This file is used to configure your project.
+# Read more about the various options under:
+# http://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files
 
-[aliases]
-test=pytest
+[metadata]
+name = whylogs
+version = 0.1.8
+description = Add a short description here!
+author = Andy Dang🤖
+author-email = andy@whylabs.ai
+license = mit
+long-description = file: README.md
+long-description-content-type = text/markdown; charset=UTF-8; variant=GFM
+url = https://github.com/pyscaffold/pyscaffold/
+project-urls =
+    Documentation = https://pyscaffold.org/
+# Change if running only on Windows, Mac or Linux (comma-separated)
+platforms = any
+# Add here all kinds of additional classifiers as defined under
+# https://pypi.python.org/pypi?%3Aaction=list_classifiers
+classifiers =
+    Development Status :: 4 - Beta
+    Programming Language :: Python
+
+[options]
+zip_safe = False
+packages = find:
+include_package_data = True
+package_dir =
+    =src
+# DON'T CHANGE THE FOLLOWING LINE! IT WILL BE UPDATED BY PYSCAFFOLD!
+setup_requires = pyscaffold>=3.2a0,<3.3a0
+# Add here dependencies of your project (semicolon/line-separated), e.g.
+install_requires =
+    click>=click==7.1.2
+    python-dateutil>=2.8.1
+    protobuf>=3.12.2
+    pyyaml>=5.3.1
+    pandas>1.0
+    marshmallow>=3.7.1
+    numpy>=1.18
+    whylabs-datasketches>=2.0.0b7
+    boto3==1.14.1
```

Comment:
I think these lines should be changed to:

```python
    boto3 >= 1.14.1
    botocore >= 1.17.44
    s3fs==0.4.2
```

As far as I know, we don't need any newer `s3fs` features from v5.x, and it's v 5.x which introduces the conflicting aiobotocore dependencies.  Additionally, this allows us to not peg the boto3/botocore dependencies which offers more flexibility as a library.  

We _really should not_ peg the boto3/botocore dependencies for a library since these are so widely used for AWS, whereas the s3fs library is less broadly used, and new features are probably less important for other libraries that a user may have installed.

Reply:
Until we move off S3fs we have to peg them!

Reply:
@andyndang, pinning to s3fs==0.4.2 removes the aiobotocore dependency.

Reply:
Oh yes! Thanks. Fixed in the next rev

Reply:
Okay, looks good to me now!